django/db/backends/mysql/schema.py

```diff
@@ -38,7 +38,12 @@ def sql_delete_check(self):
 
     @property
     def sql_rename_column(self):
-        if self.connection.mysql_is_mariadb and self.connection.mysql_version >= (10, 5, 2):
+        # MariaDB >= 10.5.2 and MySQL >= 8.0.4 supports an
```

Comment:
Isn't the `s` at the end of support redundant?
```suggestion
        # MariaDB >= 10.5.2 and MySQL >= 8.0.4 support an
```